src/components/policy/policy_regular/include/policy/policy_table/types.h

```diff
@@ -424,6 +424,9 @@ struct ConsumerFriendlyMessages : CompositeType {
 
 struct ModuleMeta : CompositeType {
  public:
+  Optional<String<0, 250> > ccpu_version;
+  Optional<String<0, 250> > language;
+  Optional<String<0, 250> > wers_country_code;
```

Comment:
wers? what is  wers?

Reply:
@LuxoftAKutsan this line came from [external policy](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/policy/policy_external/include/policy/policy_table/types.h#L490) what is wers I have no idea, but in external policy we save this value near ccpu_version and language and I decided not to separate them in regular policy

Reply:
@alexkutsan @LitvinenkoIra @ydementieiev This is the abbreviation used in FORD:
Worldwide Engineering Release System (WERS)